network-api/networkapi/mozfest/templates/fragments/blocks/mozfest_tickets_block.html

```diff
@@ -0,0 +1,31 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+    <div class="tw-bg-black tw-dark tw-py-16">
+        <div class="tw-container">
+            <h2 class="tw-mb-14">{{ title }}</h2>
+            <div class="tw-row">
+                {% for tier in self.tickets %}
+                    {% with count=self.tickets|length %}
+                        <div class="tw-mb-12 tw-w-full{% if count > 2 %} large:tw-w-1/3{% else %} large:tw-w-1/2{% endif %} tw-px-8{% if tier.sticker_text %} tw-relative{% endif %}">
```

Comment:
Do we maybe need space between the class names and the curlies? I think the Tailwind might not find the class names when they touch curlies.

Reply:
They look to output ok like this. Here's a quick test I ran:

`<h2 class="tw-mb-14{% if test_var %} tw-text-transparent{% endif %}">A title</h2>` 
output:
`<h2 class="tw-mb-14">A title</h2>`

`<h2 class="tw-mb-14{% if not test_var %} tw-text-transparent{% endif %}">A title</h2>`
output:
`<h2 class="tw-mb-14 tw-text-transparent">A title</h2>`

Reply:
I think when Tailwind generates the production stylesheet (`.css`) by parsing our template files it will see the curlies. I don't think the parsing happens against the server response but the actual file content 🤔 so it might include the curlies touching the classnames. 

I think in that case, the classname might be missing from the final stylesheet file Tailwind does not know we used the classes. They could still show up if they are used elsewhere.

Reply:
Ah ok, that might be why my test was working then. I'll tidy this up in another MR.

Reply:
https://github.com/MozillaFoundation/foundation.mozilla.org/pull/11552